core/controllers/admin.py

```diff
@@ -854,3 +854,27 @@ def post(self):
             self.render_json({})
         else:
             raise self.InvalidInputException('This app cannot send emails.')
+
+
+class InteractionsByExplorationIdHandler(base.BaseHandler):
+    """Handler to retrive the list of interactions used in an exploration."""
```

Comment:
Nit on documentation: maybe say something like "Handler for admins to retrieve..." since that distinction is useful.